vue/components/organisms/configurator/configurator.vue

```diff
@@ -291,6 +297,15 @@ export const Configurator = {
     },
     mounted: function() {
         this.bindConfigurator(this.$refs.configurator);
+
+        // in case CSR configurator is stil loading
+        // and `post_config` event was missed
+        // then re-trigger it
+        if (this.loadingCsrConfigurator) {
+            this.ripeInstance.trigger("post_config", this.ripeInstance.loadedConfig);

```

Comment:
Missing `await` here unless it's intended. If it is inteded I would add a comment to inform someone that may be checking this line in the future

Reply:
it's intended